packages/core/src/layout/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020 Ryan Pannell <r.pannell1993@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component, Fragment } from 'react';
+import { Typography, withStyles, Tooltip } from '@material-ui/core';
+import Link from '@material-ui/core/Link';
+import { Theme } from '../Page/Page';
+import MicDrop from './MicDrop';
+
+class BackButton extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <Fragment>
+      <Link onClick={this.props.history.goBack}>
+          Go back
+      </Link>... or if you think this is a bug,
+      please file an <Link href="https://github.com/spotify/backstage/issues">
+      issue</Link>.
+      </Fragment>
+    )
+  }
+}
+class ErrorPage extends Component {
```

Comment:
Gonna want these to be functional components instead, using `makeStyles`/`useStyles`. And also TypeScripts. Only if you wanna take a stab at it though, could def create issues for porting after merging too!